component-maps.yml

```diff
@@ -198,7 +199,8 @@ git:
 
   mender-connect:
     docker_image: []
```

Comment:
Same here, this should list all the same images as git:mender, except mender-client-docker.